nextvibe/configs/settings/production.py

```diff
@@ -0,0 +1,16 @@
+from .base import *
+import environ
+
+environ.Env.read_env(os.path.join(BASE_DIR, 'env', '.env.prod'))
```

Comment:
❗ `.env.prod`도 마찬가지입니다.
```suggestion
environ.Env.read_env(os.path.join(BASE_DIR, 'env', '.env.production'))
```